daemon/volumes.go

```diff
@@ -179,6 +179,18 @@ func createVolumes(container *Container) error {
 		}
 	}
 
+	// Create the dirs for BuildVolumes for backwards compatability
+	for _, volPath := range container.Config.BuildVolumes {
+		// Create the mountpoint
+		source, err := symlink.FollowSymlinkInScope(filepath.Join(container.basefs, volPath), container.basefs)
```

Comment:
Please use the wrappers `container.GetResourcePath` and `container.GetRootResourcePath` when referencing paths inside containers. Path safety is important.